tests/lazy-expression-temperature/lazy-expression-temperature.prm

```diff
@@ -0,0 +1,163 @@
+# Basic parameters file for using a function expression to define the
+# initial temperature structure using (? : ) syntax for if-else conditions.
+# 2D, Dimensional, Boussinesq, Simple Material Model
+
+# Dimension, end-time (years) and output directory
+# Only does zeroth time-step to show that initial condition works
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+set Start time                             = 0
+set End time                               = 0
+set Output directory                       = output-lazy-expression
+
+# Parameters Related to the Solver and Pressure Normalization
+# Normalization of the temperature
+set Pressure normalization                 = surface
+set Surface pressure                       = 0  
+
+# Default values are 1e-12 and 1e-7
+# Using larger values so this will just define initial condition and exit
+subsection Solver parameters
+  set Temperature solver tolerance         = 1e-6
+  
+  subsection Stokes solver parameters
+    set Linear solver tolerance            = 1e-3
+  end
+end
+
+
+# FORM OF THE EQUATIONS TO USE
+# These two choices are the same as using the Boussinesq approximation
+subsection Formulation
+  set Formulation = custom
+  set Mass conservation = incompressible
+  set Temperature equation = reference density profile
+end
+
+# GEOMETRY AND MODEL SIZE
+# Want a 2D box 5000 km wide by 1000 km deep. In METERS  5e6 x 1e6 
+# The variable repetitions divides the whole domain into 5 boxes (1000 x 1000 km)
+# as the 0th level mesh refinement: this is needed so elements are squares
+# and not elongated rectangles.
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent = 5e6
+    set Y extent = 1e6
+    set X repetitions = 5
+    set Y repetitions = 1
+  end
+end
+
+# MESH INFORMATION
+# Refine the upper 150 km of the mesh so lithosphere structure is resolved.
+# For depth of 1000km, level =7  gives element size of 1000/(2^7) = 7.8125 km
+# Total refinements = initial global + intial adaptive = 7
+# Function expression asks: if ((ymax-y<=LITH) then refinement level = 7, else
+# refinement level = 0
+subsection Mesh refinement
+  set Initial global refinement                = 3
+  set Minimum refinement level		           = 3
+  set Initial adaptive refinement              = 4
```

Comment:
Please reduce this parameter by 2 for the test. That will make the output file significantly smaller (and therefore the aspect repository faster to download).